dask/array/tests/test_array_core.py

```diff
@@ -1960,11 +1960,14 @@ def test_store_compute_false():
     at = np.zeros(shape=(4, 4))
     bt = np.zeros(shape=(4, 4))
 
-    dat, dbt = store([a, b], [at, bt], compute=False, return_stored=True)
-    assert isinstance(dat, Array) and isinstance(dbt, Array)
-    assert (at == 0).all() and (bt == 0).all()
-    assert (dat.compute() == at).all() and (dbt.compute() == bt).all()
-    assert (at == 2).all() and (bt == 3).all()
+    for load_stored in [None, False, True]:
+        dat, dbt = store(
```

Comment:
is there a way to assert that `compute` is not called in here?

Xarray has this: https://github.com/pydata/xarray/blob/65b058144ab16bd03e4b783955031e27b28d1311/xarray/tests/__init__.py#L225-L230

Reply:
I don't think that we generally tests for this, no. Adding a utility should be fine if needed

Reply:
The tests actually kind of check for this, so lets go with that.